manager/system_handler.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, list_args, query_args, filter_args, parsed_args, uri):
             .join(CveImpact, on=(CveMetadata.impact_id == CveImpact.id))
             .join(Status, on=(SystemVulnerabilities.status_id == Status.id))
             .where(SystemVulnerabilities.when_mitigated.is_null(True))
-            .where(SystemPlatform.rh_account == query_args['rh_account_number'])
+            .where(SystemPlatform.rh_account_id == query_args['rh_account_number'])
```

Comment:
this is not correct, causing a bug - no CVEs are returned for given system

Reply:
Yes, it should be the only one not replaced. Unfortunately it's not covered with tests so it was not found during development and review. I've created fixing PR #416